### PR TITLE
Lesson 2 Part 3: Create makefile

### PR DESCRIPTION
**Pull request by Derivability**
Divided source file from Lesson 2 Part 1 into multiple files and created makefile to compile them into single executable.

### CODE REVIEW

File: 01-Git_Bash_Make/Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+all : compiled
+
+compiled : guessNumber.o guessCheck.o
+	${C}  -o $@ $^
+
+guessCheck.o : guessCheck.h guessCheck.cpp
+	${C} -c $^
+
+guessNumber.o : guessNumber.cpp
```
Comment (a-vodka) [inline 01-Git_Bash_Make/Makefile:11]: I think, code for compiling guessCheck.o and guessNumber.o have to be written simultaneously. Please choose one of it.

**Comment by a-vodka** [comment]
Please, look at my code comment.

File: 01-Git_Bash_Make/Makefile (hunk 1 above)
Comment (Derivability) [reply 01-Git_Bash_Make/Makefile:11]: Should I remove last line from my makefile?

Comment (Derivability) [reply 01-Git_Bash_Make/Makefile:11]: Nevermind. My browser displayed page in weird way
изображение

**Pull request merged**